README.md

```diff
@@ -160,26 +160,13 @@ Additional reserved property names:
     * Return boolean answer: does specified `node` object have substructure match provided by `smiles_string`.
 
 ---
-
-# Results overview 
-
-## What was achieved

```

Comment:
I'm not sure why you're removing this section. It describes the capabilities that are there and seems useful.
Why do you want to remove it?

Reply:
Okay, I will return it.  I just thought it is not relevant anymore (it was a part of formal report for GSOC)